### PR TITLE
inline routine should be static here

### PR DESCRIPTION
**Pull request by imgtec**
gcc will remove this "unused" routine and cause a linking error.
symbol not found when linking.

Signed-off-by: duhuanpeng <u74147@gmail.com>

### CODE REVIEW

**Comment by FlyGoat** [comment]
LGTM so far~

File: libcpu/mips/common/mips_fpu.h [hunk 1]
```diff
@@ -18,7 +18,7 @@
  * init hardware FPU
  */
 #ifdef RT_USING_FPU
-inline void rt_hw_fpu_init(void)
+static inline void rt_hw_fpu_init(void)
```
Comment (BernardXiong) [inline libcpu/mips/common/mips_fpu.h:21]: You can use rt_inline.

File: bsp/ls1bdev/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+rtconfig.h
```
Comment (BernardXiong) [inline bsp/ls1bdev/.gitignore:1]: Please keep .gitignore file.

**Comment by imgtec** [comment]
hi, done in new patch:
- use rt_inline instead of static inline
- empty .gitignore

File: bsp/ls1bdev/.gitignore (hunk 2 above)
Comment (imgtec) [reply bsp/ls1bdev/.gitignore]: do you mean remove .gitignore in the patch?

File: libcpu/mips/common/mips_fpu.h (hunk 1 above)
Comment (imgtec) [reply libcpu/mips/common/mips_fpu.h]: done in the v2 patch(pull request)

**Comment by BernardXiong** [comment]
Please sign CLA firstly. Thank you.

**Comment by imgtec** [comment]
fail to sign CLA. let me try a new account.

**Pull request closed without merging**